config/prompts_sys.py

```
@@ -1577,12 +1577,16 @@ def get_proactive_screen_prompt(channel: str, lang: str = 'zh') -> str:
 
 
-def get_proactive_generate_prompt(lang: str = 'zh') -> str:
+def get_proactive_generate_prompt(lang: str = 'zh', music_playing_hint: str = "") -> str:
     """
     获取 Phase 2 生成阶段 prompt
     """
     lang_key = _normalize_prompt_language(lang)
-    return PROACTIVE_GENERATE_PROMPTS.get(lang_key, PROACTIVE_GENERATE_PROMPTS.get('en', PROACTIVE_GENERATE_PROMPTS['zh']))
+    prompt = PROACTIVE_GENERATE_PROMPTS.get(lang_key, PROACTIVE_GENERATE_PROMPTS.get('en', PROACTIVE_GENERATE_PROMPTS['zh']))
+    if music_playing_hint:
+        # 将提示注入到 prompt 末尾，确保 AI 能看到
```

Ruff (0.15.5)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)